sys/net/gnrc/link_layer/lorawan/include/gnrc_lorawan_internal.h

```diff
@@ -147,7 +161,11 @@ typedef struct {
  */
 typedef struct {
     uint32_t fcnt;                      /**< uplink framecounter */
-    uint32_t fcnt_down;                 /**< downlink frame counter */
+    uint32_t fcnt_down;                 /**< downlink frame counter. Mapped to network downlink frame counter if using LoRaWAN 1.1 */
+#if IS_USED(MODULE_GNRC_LORAWAN_1_1)
+    uint32_t afcnt_down;                /**< application downlink frame counter */
+    uint32_t last_fcnt_down;            /**< frame counter used to decrypt last frame. Needed to compute ConfFCnt */
+#endif
```

Comment:
here and below: please add setters/getters for these members, in order to be able to use if(IS_USED) everywhere.